docs/web-apps/automated-testing/testcafe.md

```diff
@@ -121,7 +121,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
  |TestCafe Version|Supported Browsers|
  |----|----|
- |1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.11.0)|
+ |1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.12.0)|
```

Comment:
Could you please also update the docker supported versions table on the main saucectl page: /docs/testrunner-toolkit.md? Thanks!